docker/init.bash

```diff
@@ -13,6 +13,25 @@ if [[ -z $DATABASE_URL ]]; then
     exit 1
 fi
 
+# Handle Python dependencies BEFORE attempting any `manage.py` commands
+KOBOCAT_WEB_SERVER="${KOBOCAT_WEB_SERVER:-uWSGI}"
+if [[ "${KOBOCAT_WEB_SERVER,,}" == "uwsgi" ]]; then
+    # `diff` returns exit code 1 if it finds a difference between the files
+    if ! diff -q "${KOBOCAT_SRC_DIR}/dependencies/pip/prod.txt" "${TMP_DIR}/pip_dependencies.txt"
+    then
+        echo "Syncing production pip dependencies..."
+        pip-sync dependencies/pip/prod.txt 1>/dev/null
+        cp "dependencies/pip/prod.txt" "${TMP_DIR}/pip_dependencies.txt"
+    fi
+else
+    if ! diff -q "${KOBOCAT_SRC_DIR}/dependencies/pip/dev.txt" "${TMP_DIR}/pip_dependencies.txt"
+    then
+        echo "Syncing development pip dependencies..."
+        gosu "${UWSGI_USER}" pip-sync dependencies/pip/dev.txt 1>/dev/null
```

Comment:
I know that I am the person who added the `gosu` thing but I'm wondering whether it's still useful in that case?
Otherwise, should we call it in the previous `if` block (when `uwgsi` is used)?
What do you think?

Reply:
@jnm, any feedback?

Reply:
i think you're right; we should remove it. i'm surprised it works. i'd think that files installed by pip during the docker build would be owned by root, preventing `$UWSGI_USER` from modifying them

Reply:
removed